_includes/head.html

```diff
@@ -16,4 +16,22 @@
   <link rel="manifest" href="{{ site.url }}/manifest.json">
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ site.url }}/feed.xml">
   <title>{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}</title>
-</head>
+  {% if page.layout == 'permalink' %}
+  <meta name="twitter:card" content="summary">
+  {% else %}
+  <meta name="twitter:card" content="summary_large_image">
+  {% endif %}
+  <meta name="twitter:title" content="{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}">
+  <meta name="twitter:site" content="@pwastats">
+  <meta name="twitter:creator" content="@cloudfour">
+  {% if page.title %}
+    <meta property="og:site_name" content="{{ site.title }}"/>
+  {% endif %}
+  {% if page.layout == 'permalink' %}
+  <meta property="og:title" content="{% if page.title %}{{ page.title }}{% endif %}" />
```

Comment:
I had attempted to remove the tags that I thought were the same between both Twitter and Facebook meta. Which tag(s) are you specifically referring to that are still the same?